fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -320,6 +326,7 @@ ACTOR Future<Void> getRate(UID myID,
 			batchTransactionRateInfo->setRate(rep.batchTransactionRate);
 			stats->transactionRateAllowed = rep.transactionRate;
 			stats->batchTransactionRateAllowed = rep.batchTransactionRate;
+			++stats->getRateInfo;
```

Comment:
Just to be unambiguous that this is the number of responses we get rather than the number of requests we send, can we rename this something like `getRateInfoReplies`? Or maybe we could change it to `RateUpdates` or `UpdatesFromRatekeeper`?

Reply:
sg.